SocialCareCaseViewerApi/V1/UseCase/CaseRecordsUseCase.cs

```diff
@@ -93,6 +107,7 @@ public CareCaseDataList GetResidentCases(ListCasesRequest request)
             {
                 Cases = careCaseData.ToList(),
                 NextCursor = nextCursor,
+                TotalCount = careCaseData.Count,
```

Comment:
`careCaseData` is the total with cursor and limit applied. Should this be `allCareCasedata` count instead?